test/ng/snifferSpec.js

```diff
@@ -12,14 +11,84 @@ describe('$sniffer', function() {
     return new $SnifferProvider().$get[2]($window, $document);
   }
 
+
   describe('history', function() {
     it('should be true if history.pushState defined', function() {
-      expect(sniffer({history: {pushState: noop, replaceState: noop}}).history).toBe(true);
+      var mockWindow = {
+        history: {
+          pushState: noop,
+          replaceState: noop
+        }
+      };
+
+      expect(sniffer(mockWindow).history).toBe(true);
```

Comment:
Yes, there a first commit which just makes _"the structure and layout of tests more consistent"_: https://github.com/angular/angular.js/pull/13945/commits/566b31ce7e470cbafcf6dfee38280bce45f01501